### PR TITLE
pybind/mgr/pg_autoscaler: warning message updated to mention which pool is affected

### PR DESCRIPTION
**Pull request by nithya-kumar**
This commit updates the warning message to mention which pool is affected when using PG autoscaler in "warn" mode.

Fixes: https://tracker.ceph.com/issues/42767
Signed-off-by: Nithya Kumar <nithya.rvkmeena@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by neha-ojha** [comment]
jenkins retest this please

**Comment by ljflores** [comment]
jenkins test make check

File: src/pybind/mgr/pg_autoscaler/module.py [hunk 1]
```diff
@@ -761,18 +763,29 @@ def _maybe_adjust(self) -> None:
                                    .format(p['pool_name'],
                                            p['pg_num_final'], r))
 
+        warn_pools_string = ",".join(warn_pools)
```
Comment (icyasblue) [inline src/pybind/mgr/pg_autoscaler/module.py:766]: nit, single quotes are used elsewhere in the file

Comment (ljflores) [reply src/pybind/mgr/pg_autoscaler/module.py:766]: @nithya-kumar , let me know if you need help addressing the above change!

Comment (ljflores) [reply src/pybind/mgr/pg_autoscaler/module.py:766]: Basically, you'll want to go back to your bugfix branch in your terminal and follow the same workflow to add a new commit, where you add single quotes.

See the basic workflow here: https://docs.ceph.com/en/latest/dev/developer_guide/basic-workflow/

Comment (nithya-kumar) [reply src/pybind/mgr/pg_autoscaler/module.py]: Hi @ljflores - I have fixed this and updated the PR. thanks!

Comment (ljflores) [reply src/pybind/mgr/pg_autoscaler/module.py]: @nithya-kumar great job! The only thing you'll need to fix is drop the Merge commit that is now on your branch (https://github.com/ceph/ceph/pull/48151/commits/fe3b80c8f534c3d0c13fa385c080a279e692811d). This is a really easy mistake to make when you rebase your branch, but for the future, you want to make sure to only have the regular commits on the PR, no merge commits.

To get rid of it, you can go to your bugfix branch in your terminal and run:
1. `git log`
2. Copy the SHA of the commit right before the Merge commit, i.e.:

So if your `git log` looks like this:
```
commit fe3b80c8f534c3d0c13fa385c080a279e692811d (HEAD -> ghc-osd-nithya-kumar-bug-42767, nithya-kumar/ghc-osd-nithya-kumar-bug-42767)
Merge: e1a27b70b71 9fe8b49cc06
Author: Nithya Kumar <nithya.rvkmeena@gmail.com>
Date:   Tue Sep 20 10:08:26 2022 -0400

    Merge branch 'ghc-osd-nithya-kumar-bug-42767' of https://github.com/nithya-kumar/ceph into ghc-osd-nithya-kumar-bug-42767

commit e1a27b70b710916b2331ff4b0c4701b25e450e4c (origin/ghc-osd-nithya-kumar-bug-42767)
Author: Nithya Kumar <nithya.rvkmeena@gmail.com>
Date:   Fri Sep 16 13:08:07 2022 -0700

    pybind/mgr/pg_autoscaler: warning message updated to mention which pool is affected
    
    This commit updates the warning message to mention which pool is affected when using PG autoscaler in "warn" mode.
    
    Fixes: https://tracker.ceph.com/issues/42767
    Signed-off-by: Nithya Kumar <nithya.rvkmeena@gmail.com>

commit 9fe8b49cc069d1008fc29ab93d4c1290add13aa0
Author: Nithya Kumar <nithya.rvkmeena@gmail.com>
Date:   Fri Sep 16 13:08:07 2022 -0700

    pybind/mgr/pg_autoscaler: warning message updated to mention which pool is affected
    
    This commit updates the warning message to mention which pool is affected when using PG autoscaler in "warn" mode.
    
    Fixes: https://tracker.ceph.com/issues/42767
    Signed-off-by: Nithya Kumar <nithya.rvkmeena@gmail.com>
```
You'd copy the SHA `e1a27b70b710916b2331ff4b0c4701b25e450e4c`.

3. `git reset --hard <the SHA you just copied>`
4. If you run `git log`, you should see that just the one commit is at the top now, and it should include your change with the single quotes.

In the future, you don't have to merge anything to introduce a new commit to your PR. All you'd do is:
1. Go to the bugfix branch
2. Make your change (say you made a change in file `src/pybind/mgr/pg_autoscaler/module.py`).
3. `git add src/pybind/mgr/pg_autoscaler/module.py`
4. `git commit`
5. Write and save your commit
6. `git push <the name of your remote repository> <the name of your bugfix branch>`

Then, the new commit will be cleanly added to your branch with no merge commits.

**Comment by rzarzynski** [comment]
Oops, the merge commit is still present (https://github.com/ceph/ceph/pull/48151/commits/fe3b80c8f534c3d0c13fa385c080a279e692811d).

**Comment by rzarzynski** [comment]
Dropped the merge; left 9fe8b49cc069d1008fc29ab93d4c1290add13aa0. The difference between the two commits was:

```diff
$ diff -r /tmp/1.txt /tmp/2.txt 
1c1
< commit 9fe8b49cc069d1008fc29ab93d4c1290add13aa0
---
> commit e1a27b70b710916b2331ff4b0c4701b25e450e4c
13c13
< index b7a95fc7b97..697c5719b28 100644
---
> index b7a95fc7b97..97acec20ec1 100644
36c36
< +        warn_pools_string = ",".join(warn_pools)
---
> +        warn_pools_string = ','.join(warn_pools)
```

Pushed to the already existing @nithya-kumar's branch.

**Pull request closed without merging**